python/lsst/ts/salobj/sal_info.py

```diff
@@ -752,6 +731,10 @@ async def _read_loop(self):
                             condition, reader._data_queue.maxlen
                         )
                         not reader.dds_queue_length_checker.check_nitems(len(data_list))
+                        if self.debug:
```

Comment:
Is this really necessary? Consider adding why not add a `log.debug` statement instead? if you want to skip the loop when level is larger than debug you could do:

```
if self.log.level < logging.INFO:
    ...
```

Reply:
Debugging code accidentally left in. I will remove it. (This was part of my trying to figure out why some commands were being lost -- which was the QoS issue you diagnosed).

Reply:
Fixed (and the next item). Thank you for catching this.